libs/minecraft/PosUtils.arucas

```diff
@@ -136,4 +175,249 @@ enum Direction {
         }
         return Direction.NONE;
     }
+    static fun values(){
+        return [Direction.UP, Direction.DOWN, Direction.EAST, Direction.WEST, Direction.SOUTH, Direction.NORTH];
+    }
+    static fun horizontal(){
```

Comment:
Inconsistent naming, you changed everything to `get` so this should also be `getHorizontal`, also why no `getVertical`?